tests/pdk/finfet/test_examples.py

```diff
@@ -16,7 +16,7 @@
 examples = [p.parents[0] for p in examples_dir.rglob('*.sp') \
                 if all(x not in skip_dirs for x in p.relative_to(examples_dir).parts)]
 
-@pytest.mark.parametrize( "example", examples, ids=lambda x: x.name)
+@pytest.mark.skip(reason="Refactoring in test_circuits.py")
```

Comment:
Do we still need test_examples.py or is it superseded by test_circuits.py?

Reply:
I will refactor these tests in a PR next and eliminate redundant ones.